{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40790011",
  "multipleFaultNumbers": null,
  "date": "2020-12-10T23:17:16",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-10T23:17:16",
  "information": "",
  "actualTimeOfRestoration": "2020-12-10T23:17:16",
  "outageCentrePoint": {
    "lat": 53.585895,
    "lng": -2.45644
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL1 4EN, BL1 4PZ, BL1 4QE, BL1 4QF, BL1 4QG",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}